cmd/login/runner.go

```diff
@@ -100,9 +100,11 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 		return microerror.Maskf(invalidConfigError, "Invalid number of arguments.")
 	}
 
-	// Clientcert creation if desired
-	if r.loginOptions.isWCClientCert {
-		return r.handleWCClientCert(ctx)
+	if r.loginOptions.isWC {
+		err := r.handleWCKubeconfig(ctx)
```

Comment:
Same here, the function being called is specifically for handling client certs on a workload cluster. (We are planning to move this to a different command so it'll be helpful to keep the name)

Reply:
the function for handling clientcert is still there it just moved later to avoid code duplication
https://github.com/giantswarm/kubectl-gs/pull/1102/files#diff-1f160945410f9311805f9b1ff5284e4cbb59cdd8934057db5bd7b3e40049979fR199-R271